### PR TITLE
Update for latest ouroboros-network

### PR DESCRIPTION
**Pull request by edsko**
Depends on https://github.com/input-output-hk/ouroboros-network/pull/1114 .

Should probably use the cardano repo tool to check it's using the latest dependencies of everything; I just wanted to check if I could make it work with latest consensus.

### CODE REVIEW

**Comment by edsko** [comment]
Memory profile with latest consensus (see comments on slack). 

cardano-byron-proxy

File: src/exec/Shelley.hs [hunk 1]
```diff
@@ -104,3 +109,9 @@ mkParams rr cdb nconf nstate blockchainTime = NodeArgs
   , maxUnackTxs = maxBound
   , chainSyncPipelining = pipelineDecisionLowHighMark 200 300 -- TODO: make configurable!
   }
+
+--
+-- Orphans
+--
+
+deriving via OnlyCheckIsWHNF "Peer" Peer instance NoUnexpectedThunks Peer
```
Comment (erikd) [inline src/exec/Shelley.hs]: Shouldn't this instance be with the `Peer` type? The `cardano-node` needs this same instance.

Comment (avieth) [reply src/exec/Shelley.hs]: `Peer` is defined in this module; it's not an orphan.

**Review by avieth — APPROVED** [review]
Should remove the comment about orphans but otherwise LGTM

File: src/exec/Shelley.hs (hunk 1 above)
Comment (erikd) [reply src/exec/Shelley.hs]: If its is defined in this module then the instance should be moved to near the type declaration. If its is not an orphan then `{-# OPTIONS_GHC -fno-warn-orphans #-}` should be removed.

**Comment by mrBliss** [comment]
Addressed the feedback

File: src/exec/Shelley.hs (hunk 1 above)
Comment (edsko) [reply src/exec/Shelley.hs]: Entirely possible. This was the end of a long day and I just wanted to try it and see if any changes were needed.

**Pull request merged**